core/src/test/java/org/mskcc/cbio/portal/scripts/TestImportProfileData.java

```diff
@@ -87,16 +95,33 @@ public void testImportMutationsFile() throws Exception {
         runner.run();
         //This test is to check if the ImportProfileData class indeed adds the study stable Id in front of the
         //dataset study id (e.g. studyStableId + "_breast_mutations"):
-        DaoClinicalData.deleteRecords("MUTATION_COUNT");
-        MySQLbulkLoader.flushAll();
         String studyStableId = "study_tcga_pub";
-        studyId = DaoCancerStudy.getCancerStudyByStableId(studyStableId).getInternalId();
+        CancerStudy study = DaoCancerStudy.getCancerStudyByStableId(studyStableId);
```

Comment:
This is a complex test .. maybe a comment at the start of the test function explaining what conditions are being tested and how would be helpful. The comment about whether the studyStableId is added in front or not is obsolete / misleading.

Reply:
yeah that test happens a few lines down, ill put a comment at the top that describes the entire test